helm_deploy/apply-for-legal-aid/values-uat.yaml

```diff
@@ -103,7 +103,7 @@ laa_portal:
   idpSsoTargetUrl: "https://portal.uat.legalservices.gov.uk/oamfed/idp/samlv20"
   idpSloTargetUrl: "https://portal.uat.legalservices.gov.uk/oam/server/logout"
   idpCertFingerprintAlgorithm: "<idp-cert-fingerprint-alg-goes-here>"
-  mockSaml: "true"
+  mockSaml: "false"
```

Comment:
I guess this is just for testing. remove prior to merge.

Reply:
It's in the `TEMP: Use UAT portal login, DO NOT MERGE - testing only` commit! :D